packages/http-client-csharp/generator/Microsoft.Generator.CSharp/perf/CodeWriterBenchmark.cs

```diff
@@ -17,11 +17,12 @@ public class CodeWriterBenchmark
         public CodeWriterBenchmark()
         {
             PluginInitializer.Initialize();
+            var inputModel = new InputModelType("MyModel", string.Empty, null, null, "Test model", InputModelTypeUsage.Input | InputModelTypeUsage.Output, [], null, Array.Empty<InputModelType>(), null, null, new Dictionary<string, InputModelType>(), null, false);
```

Comment:
can we convert to the factory and see if there is a way to not need the .Properties set at the end?  Like the factory just does the construction inline to hide this?